skore/tests/unit/sklearn/plot/table_report/test_estimator.py

```diff
@@ -0,0 +1,414 @@
+    labels = display.ax_.get_yticklabels()
+    assert labels[0].get_text() == "0"
+    assert labels[-1].get_text() == "4000"
+    assert display.ax_.get_ylabel() == "Count"
+    # orange
+    assert display.ax_.containers[0].patches[0].get_facecolor() == (
+        1.0,
+        0.4980392156862745,
+        0.054901960784313725,
+        0.75,
+    )
+
+    display.plot(y="gender", histplot_kwargs={"color": "blue"})
+    assert display.ax_.get_xlabel() == "Count"
+    assert display.ax_.get_ylabel() == "gender"
+    # blue
+    assert display.ax_.containers[0].patches[0].get_facecolor() == (0.0, 0.0, 1.0, 0.75)
+
+
+def test_simple_num_plots_1d(pyplot, estimator_report):
+    display = estimator_report.data.analyze(data_source="train")
+    ## for integers numeric values
+    display.plot(x="year_first_hired", histplot_kwargs={"color": "red"})
+    assert display.ax_.get_xlabel() == "year_first_hired"
+    labels = display.ax_.get_xticklabels()
+    assert labels[0].get_text() == "1970"
+    assert labels[-1].get_text() == "2010"
+    labels = display.ax_.get_yticklabels()
+    assert labels[0].get_text() == "0"
+    assert labels[-1].get_text() == "500"
+    assert display.ax_.get_ylabel() == "Count"
+    # red
+    assert display.ax_.containers[0].patches[0].get_facecolor() == (1.0, 0.0, 0.0, 0.75)
+
+    display.plot(y="year_first_hired")
+    assert display.ax_.get_xlabel() == "Count"
+    assert display.ax_.get_ylabel() == "year_first_hired"
+
+
+def test_top_k_categ_plots_1d(pyplot, estimator_report):
+    display = estimator_report.data.analyze(data_source="train")
+    display.plot(x="division")
+    assert len(display.ax_.get_xticklabels()) == 20
+    display.plot(x="division", top_k_categories=30)
+    assert len(display.ax_.get_xticklabels()) == 30
+
+
+def test_hue_plots_1d(pyplot, estimator_report):
+    display = estimator_report.data.analyze(data_source="train")
+    display.plot(x="gender", hue="current_annual_salary")
+    assert "BoxPlotContainer" in display.ax_.containers[0].__class__.__name__
+    legend_labels = display.ax_.legend_.texts
+    assert legend_labels[0].get_text() == "50000"
+    assert legend_labels[-1].get_text() == "300000"
+    assert display.ax_.legend_.get_title().get_text() == "current_annual_salary"
+
+    display.plot(y="gender", hue="current_annual_salary")
+    assert "BoxPlotContainer" in display.ax_.containers[0].__class__.__name__
+
+    msg = "If 'x' and 'y' are categories, 'hue' must be continuous"
+    with pytest.raises(ValueError, match=msg):
+        display.plot(x="gender", hue="division", top_k_categories=30)
+
+    display.plot(y="year_first_hired", hue="current_annual_salary")
+    assert display.ax_.get_xlabel() == "current_annual_salary"
+    assert display.ax_.get_ylabel() == "year_first_hired"
+    assert display.ax_.legend_.get_title().get_text() == "current_annual_salary"
+
+
+def test_1D_plot_with_duration_data(display):
```

Comment:
```suggestion
def test_1D_plot_with_duration_data(pyplot, display):
```

It is quite important to add the `pyplot` pytest fixture because it will be in charge of closing figure once the test is run. It avoids weird behaviour where figure are shown to developer when running the test suite.

So in general, if a test contains a `.plot` call, you need to add this `pyplot` in the signature.